tests/stickers/createGuildSticker.test.ts

```diff
@@ -0,0 +1,16 @@
+import { assertEquals } from "../deps.ts";
+import { loadBot } from "../mod.ts";
+import { CACHED_COMMUNITY_GUILD_ID } from "../utils.ts";
+
+Deno.test("[stickers] Create guild sticker", async () => {
+  const bot = loadBot();
+  const sticker = await bot.helpers.createGuildSticker(CACHED_COMMUNITY_GUILD_ID, {
+    name: "sticker name",
+    description: "sticker description",
+    tags: "sticker tags",
+    file: { blob: new Blob(), name: "sticker file name" },
```

Comment:
Should this not fail if the filname doesnt have a valid file suffix like .png

Reply:
@Skillz4Killz we do not have any emoji tests so there's no image/asset i can reuse to test
any ideas?